src/main/java/org/carlmontrobotics/commands/AimArmSpeaker.java

```diff
@@ -6,24 +6,27 @@
 
 import org.carlmontrobotics.subsystems.Arm;
 import org.carlmontrobotics.subsystems.Limelight;
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.wpilibj.Timer;
 import edu.wpi.first.wpilibj2.command.Command;
 
 public class AimArmSpeaker extends Command {
   private final Arm arm;
-  private final Limelight ll; //Shooter ll
+  private final Limelight ll;
 
   /** Creates a new AimOuttakeSpeaker. */
   public AimArmSpeaker(Arm arm, Limelight ll) {
     // Use addRequirements() here to declare subsystem dependencies.
     addRequirements(this.arm = arm);
-    this.ll = ll;
+    addRequirements(this.ll = ll);
```

Comment:
This will cause other commands that use the limelight to be cancelled when this command starts. I don't think you want that.